### ISSUE TITLE
[a11y] make challenge description scrollable 

### ISSUE THREAD

**Issue by PickleNik** [typehero/typehero#244]
Instead of `leftWrapper.tsx` layout having `overflow-y-auto` with no scrollbar (because scrollbar pushed the entire left side UI with tabs which looked ugly), have `<Description/>` component itself be scrollable.

For that it's height needs to be explicitly set to `container (leftWrapper) height` - `comments/index.tsx height` - `tabs height` and comments should be positioned `absolute`ly instead of `sticky` so that when it expands it does not affect description.

Do the same for Submissions and Solutions `left-wrapper container height` - `tabs height`
